### PR TITLE
Update and refresh 'Sponsorships' page 

### LINKED ISSUE #896: Make getmonero.org/community/sponsorships list what the sponsors did
*Issue originally created by @OsrsNeedsF2P*

*This issue was created on gitlab and then migrated here. Only the original post was migrated, not the comments. Please take a look at the discussions on the original Gitlab issue before commenting here: https://repo.getmonero.org/monero-project/monero-site/-/issues/981*

---
This was brought up in the Monero Coffee Chat (Aug 24th), and some ideas were thrown around about how it would be done. Discuss?

### LINKED ISSUE #1168: Remove unactive sponsors and add active ones in the "Sponsorships" page
Let's also consider #896 when making the change.

### PR DESCRIPTION
**Pull request by erciccione**
Update and refresh 'Sponsorships' page

- Removed MyMonero, JetBrains, Dome9, Kitware, Navicat and Araxis. Added Tari Labs, Globee and Macstadium
- Used bigger boxes for bigger contributors (currently Tari Labs and Globee)
- Removed redundant titles, improved styiling and added details about what is being sponsored
- Added sanitised referrer policy for all external links
- Added logos for Globee, Tari Labs and Macstadium
- Moved images to their own folder in /img/sponsors

Based on #1169 
Closes #896 
Closes #1168 

preview:
sponsor

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -114,7 +114,12 @@ merchants:
   onion: Onion address (needs Tor)
 
 sponsorships:
-  intro: The following businesses have supported the Monero Project in its goal to bring financial privacy to the world. We couldn't be more grateful for their contributions. If you would like to sponsor the Monero Project and be listed on this page, please send an email to dev@getmonero.org.
+  intro: The following businesses actively support the Monero Project in its goal to bring financial privacy to the world. We couldn't be more grateful for their contributions. If you would like to sponsor the Monero Project and be listed on this page, please send an email to dev@getmonero.org.
+  tari: Tari Labs sponsors the CDN used by this website, for a cost of $5150 a month.
+  globee: GloBee sponsors the main Monero website infrastructure, for a cost of $2000 a month.
+  symas: Symas sponsors its CTO Howard Chu (hyc) for his work on the Monero codebase, specifically to database-related code.
+  forknetworking: Fork Networking provides a server which is used for development.
+  macstadium: Macstadium sponsors a hosted Mac Mini that is used for Mac-related development.
```
Comment (selsta) [inline _i18n/en.yml:122]:
```suggestion
  macstadium: MacStadium sponsors a dedicated Mac Mini that is used for Mac-related development.
```

**Comment by selsta** [comment]
Is it possible to vertically center the "Fork Networking" image / text?

File: css/custom.css [hunk 2]
```diff
@@ -4218,8 +4218,16 @@ p.hangouts-social {
...
+}
+
+.sponsors p#tari {
+    padding-top: 3rem;
+}
+
 .sponsors img {
-    width: 100%;
+    max-height: 100px;
 }
```
Comment (l33d4n) [inline css/custom.css:4232]:
```suggestion
.sponsors .info-block {
    display: flex;
    justify-content: center;
    align-items: center;
}
```

**Review by l33d4n — COMMENTED** [review]
## Before:

image

## After:

image

File: css/custom.css (hunk 2 above)
Comment (selsta) [reply css/custom.css:4232]: Thanks, that was what I was looking for :)

Comment (erciccione) [reply css/custom.css:4238]: Thanks @l33d4n. Added.

**Comment by erciccione** [comment]
Verticalli aligned blocks and edited symas' description (ass suggested by @hyc in https://github.com/monero-project/monero-site/issues/896#issuecomment-685776667)

File: _i18n/en.yml (hunk 1 above)
Comment (erciccione) [reply _i18n/en.yml]: Done.

**Review by l33d4n — APPROVED** [review]

**Comment by erciccione** [comment]
Rebased

**Comment by selsta** [comment]
$5150 is a bit weird, I doubt CDN prices are static, how about simply 5000? :P

**Comment by erciccione** [comment]
That's the amount fluffypony reported. I think he has direct access to those bills so he knows the specific amount, wouldn't change to a generic number when we have a specific amount reported :)

**Comment by erciccione** [comment]
Adapted to #1145 and fixed a link.

Note that this PR should be merged after #1169.

**Comment by erciccione** [comment]
Rebased

**Review by selsta — APPROVED** [review]

**Pull request merged**